fe/fe-core/src/main/java/com/starrocks/catalog/MysqlTable.java

```diff
@@ -235,4 +235,9 @@ public boolean isSupported() {
     public boolean supportInsert() {
         return true;
     }
+
+    @Override
+    public boolean supportTimeTravel() {
+        return true;
```

Comment:
Really?

Reply:
sure.  https://github.com/StarRocks/starrocks/commit/4f5cdded0301229605d5c231b4c221c19752a747. only used for mysql external table